sympy/utilities/lambdify.py

```diff
@@ -297,7 +297,18 @@ def lambdify(args, expr, modules=None, printer=None, use_imps=True,
     """
     from sympy.core.symbol import Symbol
     from sympy.utilities.iterables import flatten
-
+    invalid_str_expr = 0
+    invalid_str_args = 0
+    if isinstance(expr, str):
+        if not tuple(expr.replace(',','')) == tuple(expr.split(',')):
```

Comment:
You can merge the two ```if```s like this:
```if isinstance(expr, str) and not tuple(expr.replace(',','')) == tuple(expr.split(',')):```
I suspect that that's going to be too long for one line (80 characters is the limit, some people like to stop at 72). I'm not sure whether a two-line condition is going to be easier to read than nested ```if```s; if it isn't, this can be left as is.